public/aside.php

```diff
@@ -135,7 +135,7 @@
             </div>
         </section>
     <?php endif; ?>
-    <?php if ($this->options->JAside_3DTag === 'on') : ?>
+    <?php if ($this->options->JAside_Tag !== 'off') : ?>
```

Comment:
`<?php if ($this->options->JAside_Tag !='' && $this->options->JAside_Tag !== 'off') : ?>`